app/Mage.php

```diff
@@ -12,45 +12,49 @@
 
 define('DS', DIRECTORY_SEPARATOR);
 define('PS', PATH_SEPARATOR);
-if (file_exists(__DIR__ . '/vendor/mahocommerce/maho')) {
-    define('BP', dirname(getcwd()));
-} elseif (str_contains(__DIR__, '/vendor/mahocommerce/maho')) {
-    define('BP', str_replace('/vendor/mahocommerce/maho/app', '', __DIR__));
+define('BP', str_replace('/vendor/mahocommerce/maho', '', dirname(__DIR__)));
+
+if (file_exists(BP . '/vendor/mahocommerce/maho')) {
+    define('MAHO_IS_STARTER_KIT', true);
+    define('MAHO_ROOT_SOURCE_DIR', BP . '/vendor/mahocommerce/maho');
```

Comment:
thank you @justinbeaty for all your amazing work, in the past couple of days I was super busy with personal stuff so I'm sorry I can't comment only briefly.

```suggestion
    define('MAHO_IS_CHILD_PROJECT', true);
    define('MAHO_FRAMEWORK_DIR', BP . '/vendor/mahocommerce/maho');
```

the concept of child project or child theme is borrowed from wordpress and it seems to me a bit easier to understand

do you like them?

Reply:
Yeah, that seems like better naming. Don't worry if you can't get to review it for a while.